.env.example

```diff
@@ -1,5 +1,6 @@
 ETHERSCAN_API_KEY=""
 PRIVATE_KEY=""
+PUBLIC_ADDRESS=""
```

Comment:
@harshit-wq Why public address?

Reply:
It was needed for testing purposes, IF we want to test it using different private key and public address we can do so by just changing the parameters in .env file but the private key and public address should be of same account. You can have a look at the use of public address in this file ([here](https://github.com/harshit-wq/dsa-connectors/blob/58818525ac36b0957a7dfdb7f356ccf4c524a302/test/mainnet/permit_erc20/test-aave.js#L84)).

If needed i can try to remove that but i will have to hardcode my public address in the test file but then the testcase fails if private key and public address is not of same account.

Reply:
@thrilok209

Reply:
But you can just mention this in testing, and remove it from `.env`

Reply:
Okay, so I will remove it and hardcode my public address in the test file with the comment specifying the needful. @pradyuman-verma

Reply:
Made the changes in the new commit

Reply:
https://github.com/harshit-wq/dsa-connectors/blob/harshit_permit_ERC20/.env.example

Reply:
okay!